dff/core/engine/conditions.py

```diff
@@ -79,23 +77,25 @@ def check_cond_seq(cond_seq: list):
 
 
 _any = any
-""" _any is an alias for any. """
+"""
+_any is an alias for any.
+"""
 _all = all
-""" _all is an alias for all. """
+"""
+_all is an alias for all.
+"""
 
 
 @validate_arguments
 def aggregate(cond_seq: list, aggregate_func: Callable = _any, *args, **kwargs) -> Callable:
```

Comment:
Should we add Callable signature (☝️): `Callable[[Context, Actor, Any, Any], bool]` (AFAIK) here.